usm_wrappers/salt_wrapper.py

```diff
@@ -138,6 +195,48 @@ def _check_ip_network(minion, ip, ip_list, network, subnets,
     return public_network, cluster_network
 
 
+def get_minion_disk_info(minions):
+    columes = 'NAME,KNAME,FSTYPE,MOUNTPOINT,UUID,PARTUUID,MODEL,SIZE,TYPE,' \
```

Comment:
A doc string would go a long way toward describing what this function expects to return.